Dockerfile

```diff
@@ -30,6 +30,5 @@ EXPOSE 8009
 RUN mkdir /usr/local/tomcat/home/
 
 # Launch Tomcat
-CMD cp /tmp/cpm-props/*.properties /usr/local/tomcat/home/; cp /tmp/tomcat/server.xml /usr/local/tomcat/conf/; cp /tmp/jdbc-driver/* /usr/local/tomcat/lib/; catalina.sh run
-#CMD cp /tmp/cpm-props/* /usr/local/tomcat/home/; cp /tmp/jdbc-driver/* /usr/local/tomcat/lib/; catalina.sh run
-#CMD /bin/bash
+# OpenShift secrets assumed mounted as /secrets/${secret}
```

Comment:
What's the value of ${secret}?
Seems the /secrets/ folder contains both subfolers: /app and /tomcat.

Reply:
That comment was just intended to provide additional context.
For consistency, ${secret} should match the name of the secrets in each OpenShift project/namespace.  
Secrets "app" and "tomcat" should be mounted as /secrets/app and /secrets/tomcat respectively.